CONTRIBUTING.md

```diff
@@ -127,3 +151,13 @@ master constitutes a tagged release.
 - merge hotfix-vX.X.X to master
 - merge hotfix-vX.X.X to develop
 - delete the hotfix-vX.X.X branch
+
+
+## Testing
+
+All repos should be hooked up to [CircleCI](https://circleci.com/).
```

Comment:
This seems odd for contributing.md. (It reads to me as your fork should have circle CI testing)

I think it was intended to mean that all of tendermint organization's repos must have commits passing circle CI tests before merging.

Reply:
That's right. That's why I moved it to the bottom and will probably break it out all together